inst/include/child_model_simulation.hpp

```diff
@@ -1383,12 +1383,10 @@ void fill_model_outputs(int time_step,
         }// end hcTT
 
         for (int dur = 0; dur < ss.hTS; ++dur) {
-          if (state_next.children.hc1_art_pop(0, hd, a, s) > 0) {
```

Comment:
The issue here is `a`. The loop run `s` up to `pars.base.options.p_idx_fertility_first` which is 15. But `hc1_art_pop`'s 3rd dimension is age which runs up to `hc1AG` which is `5` so we were reading off the end of the data here.